### DISCUSSION TITLE
Paddle OCR install error

### DISCUSSION THREAD

**Discussion by VanQuyen02** [PaddlePaddle/PaddleOCR#12228] [Q&A]
When I clone PaddleOCR on colab and "pip install -e . ".This error  happens:
/content/drive/MyDrive/PaddleOCR
Obtaining file:///content/drive/MyDrive/PaddleOCR
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  Preparing metadata (setup.py) ... error
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
How to fix this error

**Comment by said46** [comment]
Same issue, tried all the recommendations from [here](https://www.easeus.com/data-recovery/python-setup-py-egg-info-failed-with-error-code-1.html) and [here](https://stackoverflow.com/questions/35991403/pip-install-unroll-python-setup-py-egg-info-failed-with-error-code-1).

**Comment by SWHL** [comment]
Please try to pip install PaddleOCR by the following and try again:
```bash
pip install paddleocr
```

**Reply by shoou** [reply]
it's should be `pip install paddleocr`

**Reply by SWHL** [reply]
Tks, it's a typo.
